libs/common/src/helpers/is-ukef-email.test.ts

```diff
@@ -0,0 +1,40 @@
+import { isUkefEmail } from './is-ukef-email';
+
+describe('isUkefEmail', () => {
+  const invalidEmails = [
+    '',
+    ' ',
+    'test',
+    'test@',
+    'test@example.com',
+    'test@gov.uk',
+    'maker1@ukexportfinance.com',
+    'maker1@ukexportfinance.co.uk',
+    'admin@ukexportfinance.com',
+    'admin@ukexportfinance.co.uk',
+    'maker1@ukef.co.uk',
```

Comment:
Maybe should add `maker1@ukef.gov.uk` as well

Reply:
Have added.